payments-core/src/main/java/com/stripe/android/model/ConsumerSignUpConsentAction.kt

```diff
@@ -0,0 +1,8 @@
+package com.stripe.android.model
+
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+enum class ConsumerSignUpConsentAction(val value: String) {
+    Checkbox("checkbox_mobile"), Button("button_mobile")
```

Comment:
Let's change this to `clicked_checkbox_mobile` and `clicked_button_mobile` instead 

I've updated the original [doc](https://paper.dropbox.com/doc/Link-consent_action-types--Bn2PpoxuuzsuyvoA0a2U2L89Ag-GDODEncdIhQ2ST7W72ZNj#:uid=204286646721432093305479&h2=Mobile-SDK---iOS/android) for these values.
Thanks!